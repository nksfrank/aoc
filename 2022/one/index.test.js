import fs from "node:fs";
import path from "node:path";
import { expect, test } from "vitest";
import { partOne, partTwo } from ".";

const data = `1000
2000
3000

4000

5000
6000

7000
8000
9000

10000`;

test("part one works", () => {
  expect(partOne(data)).toEqual(24000);
});
test("part two works", () => {
  expect(partTwo(data)).toEqual(45000);
});