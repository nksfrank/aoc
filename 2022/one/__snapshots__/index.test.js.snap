// Vitest Snapshot v1

exports[`live 1`] = `72070`;

exports[`live 2`] = `211805`;
